File: ImageApp2/app/src/main/java/com/example/imageapp2/MainActivity.java
package com.example.imageapp2;

import androidx.activity.result.ActivityResult;
import androidx.activity.result.ActivityResultLauncher;
import androidx.activity.result.contract.ActivityResultContracts;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import com.google.android.gms.cast.framework.media.ImagePicker;
import com.google.android.material.floatingactionbutton.FloatingActionButton;

public class MainActivity extends AppCompatActivity {
    ImageView coverImg;
    FloatingActionButton flb;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        coverImg = findViewById(R.id.coverImg);
        flb = findViewById(R.id.floatingActionButton);
        flb.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
//
            }
        });
    }
}

File: MVVMmodel/app/src/main/java/com/example/mvvmmodel/MainActivity.kt
package com.example.mvvmmodel

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.example.mvvmmodel.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

         var activityMainBinding : ActivityMainBinding = DataBindingUtil.setContentView(this,R.layout.activity_main)

        var loginViewModel1 : LoginViewModel
        activityMainBinding.loginViewModel
    }
}